pkg/utils/tso.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"time"
+)
+
+// TSOToTime translates tso to time
+func TSOToTime(ts int64) time.Time {
+	t := time.Unix(ts>>18/1000, 0)
```

Comment:
what about to be more precise get the Millisecond part to

Reply:
I think @WangXiangUSTC want a fuzzy time here, I would add comments or change it.  But anther problem, do we need a time zone in `generate savepoint`?

Reply:
still need time zone, we can retain the code now, I will refine it next pr.

Reply:
I refine it later